src/components/ChatBotMessages/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import ChatBot from "react-simple-chatbot";
+import { ThemeProvider } from "styled-components";
+import { useTranslation } from "react-i18next";
+import womanBot from "./images/woman.svg";
+import womanSeeksHelp from "./images/womanSeeksHelp.svg";
+import "./index.scss";
+import { Redirect } from "react-router-dom";
+
+const CHATBOT_THEME = {
+  background: "#FFFEFC",
+  fontFamily: "Roboto",
+  headerBgColor: "#FFBFB5",
+  headerFontColor: "#fff",
+  headerFontSize: "15px",
+  botBubbleColor: "#C8D7C2",
+  botFontColor: "#fff",
+  userBubbleColor: "#FFBFB5",
+  userFontColor: "#fff",
+};
+
+const GoToSearchHelpPage = () => {
+  const { t } = useTranslation();
+  return (
+    <div>
+      <p>{t("chatBotMessages.goToSeekHelpPageText")}</p>
+      <Redirect to="/seekhelp" />
+    </div>
+  );
+};
+
+const GoToOpportunitiesPage = () => {
+  const { t } = useTranslation();
+  return (
+    <div>
+      <p>{t("chatBotMessages.goToOpportunitiesPageText")}</p>
+      <Redirect to="/opportunities" />
+    </div>
+  );
+};
+
+const GoToBlogPage = () => {
+  const { t } = useTranslation();
+  return (
+    <div>
+      <p>{t("chatBotMessages.goToBlogPageText")}</p>
+      <Redirect to="/blog" />
+    </div>
+  );
+};
+
+const ChatBotMessages = () => {
+  const { t } = useTranslation();
+
+  // You can find chatbot documentation here https://lucasbassetti.com.br/react-simple-chatbot/#/
+  const steps = [
+    {
+      id: "1",
+      message: t("chatBotMessages.welcomeMessage"),
+      trigger: "2",
+    },
+    {
+      id: "2",
+      options: [
+        { value: 1, label: t("chatBotMessages.helpLabel"), trigger: "3" },
+        { value: 2, label: t("chatBotMessages.findLabel"), trigger: "4" },
+        { value: 3, label: t("chatBotMessages.blogLabel"), trigger: "5" },
+        {
+          value: 4,
+          label: t("chatBotMessages.opportunityLabel"),
+          trigger: "6",
+        },
+      ],
+    },
+    {
+      id: "3",
+      component: (
+        <div>
+          <p>
+            {t("chatBotMessages.immediateHelpMessagePt1")}
```

Comment:
1. if you do this, you should be 0-indexing it (code doesnt start counting from 1)
2. we don't do translations like this with pt0 pt1. it makes it hard to change the translations and it makes it hard for translators to understand what is going on with the gap between the parts

https://www.i18next.com/translation-function/interpolation

Reply:
I tried several ways on adding anchor in the middle of sentence, but none of the methods worked. There was a way to add dangerouslySetInnerHTML, but I preferred to change sentence structure and not use it.

Reply:
dangerouslySetInnerHtml would be permissible here.

and using this section
https://www.i18next.com/translation-function/interpolation#unescape

Reply:
I tried this but it didn't work and then I found this thread https://github.com/i18next/i18next/issues/1112

Reply:
I did it like shown in the example in the link you sent and I tried to used <Trans /> hook, but both ways didn't work. I might be missing something, but I am not sure what